xla/service/gpu/triton_support.cc

```diff
@@ -628,6 +628,13 @@ absl::Status EnsureTritonSupportsComputeCapability(
     const se::GpuComputeCapability& gpu_compute_capability) {
   auto cuda_compute_capability =
       std::get_if<se::CudaComputeCapability>(&gpu_compute_capability);
+  auto rocm_compute_capability =
```

Comment:
This file is in flux and for the time being the support is expressed in two places:

- Here, where you modified the code to add support. This is the support for everything except MatMul.
- In the `legacy_triton` namespace above, where support for MatMul is defined. Consider updating functions in that namespace as well.

Reply:
Also, changes to this file normally mean that you have to modify the `triton_support_test.cc` file as well. Please add RoCM testing to that file.

Reply:
There are some ROCM support in functions already, e.g.:
[CanTritonHandleGEMM](https://github.com/openxla/xla/blame/c2d607e21ab93d3bd20cfd9e781bcec79c9b925f/xla/service/gpu/triton_support.cc#L241C17-L241C36)
https://github.com/openxla/xla/blame/c2d607e21ab93d3bd20cfd9e781bcec79c9b925f/xla/service/gpu/triton_support.cc#L450C28-L450C28

Are you referring to this or some additional support should be implemented?


Also, there is some rocm support in triton_support_test.cc:
https://github.com/openxla/xla/blob/c2d607e21ab93d3bd20cfd9e781bcec79c9b925f/xla/service/gpu/triton_support_test.cc#L136

Reply:
Those are the ones I was referring to. If you think the existing coverage is enough, that's OK with me.

Reply:
Thank you for clarification.